# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/fuse2/hotfix-closefrom.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Hotfix: only define closefrom when not using glibc as it
already exists and has a conflicting type.

NoTag
--- fuse-2.9.9/util/ulockmgr_server.c.vanilla	2023-01-19 15:24:06.900537152 +0100
+++ fuse-2.9.9/util/ulockmgr_server.c	2023-01-19 15:36:12.563577111 +0100
@@ -124,6 +124,7 @@
 	return res;
 }
 
+#ifndef __GLIBC__
 static int closefrom(int minfd)
 {
 	DIR *dir = opendir("/proc/self/fd");
@@ -141,6 +142,7 @@
 	}
 	return 0;
 }
+#endif
 
 static void send_reply(int cfd, struct message *msg)
 {
